### DISCUSSION TITLE
Install open62541pp with vcpkg

### DISCUSSION THREAD

**Discussion by lukasberbuer** [open62541pp/open62541pp#196] [Announcements]
open62541pp can now be built, installed and integrated easily with the package manager [vcpkg](https://vcpkg.io):

```shell
vcpkg install open62541pp
```

Please refer to the [vcpkg documentation](https://vcpkg.io/en/getting-started) how to use it within your project.

Thanks @traversaro for the contribution 🚀

**Comment by ozlb** [comment]
[`open62541pp_vcpkg`](https://github.com/ozlb/open62541pp_vcpkg) Repository provides precompiled ready‑to‑use static builds.
